actions/trigger-argo-workflow/cmd/trigger-argo-workflow/main.go

```diff
@@ -173,7 +214,7 @@ func run(c *cli.Context, level *slog.LevelVar, logger *slog.Logger) error {
 		extraArgs = append(extraArgs, "--parameter", param)
 	}
 
-	extraArgs = append(extraArgs, c.Args().Tail()...)
+	extraArgs = append(extraArgs, c.Args().Slice()...)
```

Comment:
This one caught my eye. Looks like `Tail()` is still available, so is this a fix for a bug?

Reply:
Yes, this is a result of the `submit` command being an actual command.